.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        gdal_version: ["2.4.3", "3.1.2"]
+        openjdk_version: [8, 11]
+        openjpeg_version: ["2.3.0"]
+        variant: ["slim"]
+        include:
+          - gdal_version: "2.4.3"
+            libproj_version: "5.2.0"
+          - gdal_version: "3.1.2"
+            libproj_version: "7.1.0"
+    env:
+      DOCKER_BUILDKIT: 1
+      GDAL_VERSION: ${{ matrix.gdal_version }}
+      OPENJDK_VERSION: ${{ matrix.openjdk_version }}
+      OPENJPEG_VERSION: ${{ matrix.openjpeg_version }}
+      LIBPROJ_VERSION: ${{ matrix.libproj_version }}
+      VARIANT: ${{ matrix.variant }}
```

Comment:
Right now we're only building the `slim` variant. Should we delete the Alpine Dockerfile template? I left it there in case somebody wants to use it to build locally, but it's effectively untested and I would push to remove it.